README.md

```diff
@@ -96,11 +92,11 @@ In order to run this project you need:
 
```

Comment:
- [ ] [OPTIONAL]
Kindly include appropriate instructions in your ReadMe on how to run your app, under the _Getting Started_ section.